File: C C++/Set-14-2021/tarefa.c
#include<stdio.h>

int main()
{
    int n1,n2,r1,r2;
    printf("Digite n1: ");
    scanf("%d", &n1);
    printf("Digite n2: ");
    scanf("%d", &n2);

    if (n2 == 0){
        printf("Nao e possivel dividir por zero");
    }

    if (n1 % 3 == 0 && n1 % 5 == 0){
        printf("O número %d é divisível por 3 e 5\n", n1);
    }
    else{
        printf("O número %d não é divisível por 3 e 5 ao mesmo tempo\n", n1);
    }
    if (n2 % 3 == 0 && n2 % 5 == 0){
        printf("O número %d é divisível por 3 e 5\n", n2);
    }
    else{
        printf("O número %d não é divisível por 3 e 5 ao mesmo tempo\n", n2);

    if (n1 > 0){
        printf("O número n1 = %d é positivo", n1);
        if (n1 % 4 == 0){
            printf(" e múltiplo de 4");
        }
        else{}
    }
    else if (n1 < 0){
        printf("\nO número n1 = %d é negativo", n1);
        if (n1 % 4 == 0){
            printf(" e múltiplo de 4");
        }
        else{}
    }
    else{
        printf("\nO número n1 é zero", n1);
    }
    if (n2 > 0){
        printf("\nO número n2 = %d é positivo", n2);
        if (n2 % 4 == 0){
            printf(" e múltiplo de 4");
        }
        else{}
    }
    else if (n2 < 0){
        printf("\nO número n2 = %d é negativo", n2);
        if (n2 % 4 == 0){
            printf(" e múltiplo de 4");
        }
        else{}
    }
    else{
        printf("\nO número n1 é zero", n2);
    }
    }
}

File: Set-14-2021/tarefa.py
n1 = int(input("Digite n1: "))
n2 = int(input("Digite n2: "))

if (n2 == 0):
    print("Não é possível dividir por zero.")

if (n1 % 3 == 0 and n1 % 5 == 0):
    print("O número n1 = {} é divisível por 3 e 5.".format(n1))
else:
    print("O número n1 = {} não é divisível por 3 e 5 ao mesmo tempo".format(n1))

if (n2 % 3 == 0 and n2 % 5 == 0):
    print("O número n2 = {} é divisível por 3 e 5.".format(n2))
else:
    print("O número n2 = {} não é divisível por 3 e 5 ao mesmo tempo".format(n2))

if (n1 > 0):
    print("O número n1 = {} é positivo".format(n1))
    if (n1 % 4 == 0):
        print("e divisível por 4")
elif (n1 < 0):
    print("O número n1 = {} é negativo".format(n1))
    if (n1 % 4 == 0):
        print("e divisível por 4")
else:
    print("O número n1 é zero.")

if (n2 > 0):
    print("O número n2 = {} é positivo".format(n2))
    if (n2 % 4 == 0):
        print("e divisível por 4")
elif (n2 < 0):
    print("O número n2 = {} é negativo".format(n2))
    if (n2 % 4 == 0):
        print("e divisível por 4")
else:
    print("O número n2 é zero.")